source/fundamentals/crud/run-command.txt

```diff
@@ -63,7 +67,10 @@ Information <addl-info-runcommand>`.
 Response
 --------
 
-Each method returns a document or a cursor containing documents with the
+Each method returns a result or a cursor that contains the response
+from the database after you run a command. Each database command
```

Comment:
Issue: "after you run a command" is a little vague

Suggestion: I know we try to avoid passive, but I think it makes sense here: "...from the database after the command has completed." (or something similar)